index.html

```diff
@@ -0,0 +1,222 @@
+    <!--Google-Fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <!--Font Awesome-->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="left_side">
+        <div class="profileText">
+          <div class="imgBox">
+            <img src="./images/profile.jpeg" alt="" />
+          </div>
+          <h2>Mahimkar Saad<br /><span>Web Developer</span></h2>
+        </div>
+        <div class="contactInfo">
+          <h3 class="title">Contact Info</h3>
+          <ul>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-phone" aria-hidden="true"></i
+              ></span>
+              <span class="text">8888**44**</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-envelope-o" aria-hidden="true"></i
+              ></span>
+              <span class="text">saadmahimkar@icloud.com</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-globe" aria-hidden="true"></i
+              ></span>
+              <span class="text">LoremIpsum.com</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-github" aria-hidden="true"></i
+              ></span>
+              <span class="text">github.com/SaadMahi</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-map-marker" aria-hidden="true"></i
+              ></span>
+              <span class="text">Navi Mumbai, Kharghar</span>
+            </li>
+          </ul>
+        </div>
+
+        <div class="contactInfo education">
+          <h3 class="title">Education</h3>
+          <ul>
+            <li>
+              <h5>2021 - 2024</h5>
+              <h4>Diploma in Computer Engineering</h4>
+              <h4>A.R. KALSERKAR, Polytechnic</h4>
+            </li>
+            <li>
+              <h5>2021</h5>
+              <h4>Integrated Digital Marketing</h4>
+              <h4>FREELANCERS ACADEMY</h4>
+            </li>
+            <li>
+              <h5>2022 - present</h5>
+              <h4>HTML & CSS</h4>
+              <h4>nextGenDevTech</h4>
+            </li>
+          </ul>
+        </div>
+
+        <div class="contactInfo-language">
+          <h3 class="title">Languages</h3>
+          <ul>
+            <li>
+              <span class="text">English</span>
```

Comment:
Use some light colors for languages text.

Reply:
light color applied.